src/cmd_definitions.c

```diff
@@ -8,6 +8,7 @@
 #define COUNT_OF(arr) (sizeof(arr) / sizeof(arr[0]))
 
 const cmd_definition cmd_definitions[] = {
+    {.name = "help", .arg_count = 0, .executor = &cmd_help},
```

Comment:
Change `cmd_definition` to add a `char *description` member and fill `.description = "my super description"` for each entry.